docs/zh-cn/1.2.0/user_doc/hardware-environment.md

```diff
@@ -0,0 +1,48 @@
+# 软硬件环境建议配置
+
+DolphinScheduler 作为一款开源分布式工作流任务调度系统，可以很好的部署和运行在 Intel 架构服务器环境及主流虚拟化环境下，并支持主流的Linux操作系统环境。
+
+## 1. Linux 操作系统版本要求
+
+| 操作系统       | 版本         |
+| :----------------------- | :----------: |
+| Red Hat Enterprise Linux | 7.0 及以上   |
+| CentOS                   | 7.0 及以上   |
+| Oracle Enterprise Linux  | 7.0 及以上   |
+| Ubuntu LTS               | 16.04 及以上 |
+
+> **注意：**
+>以上 Linux 操作系统可运行在物理服务器以及 VMware、KVM、XEN 主流虚拟化环境上。
+
+## 2. 服务器建议配置
+DolphinScheduler 支持运行在 Intel x86-64 架构的 64 位通用硬件服务器平台。对生产环境的服务器硬件配置有以下建议：
+### 生产环境
+
+| **CPU** | **内存** | **硬盘类型** | **网络** | **实例数量** |
+| --- | --- | --- | --- | --- |
+| 4核+ | 8 GB+ | SAS | 千兆网卡 | 1+ |
+
+> **注意：**
+> - 以上建议配置为部署 DolphinScheduler 的最低配置，生产环境强烈推荐使用更高的配置。
+> - 硬盘大小配置建议 50GB+ ，系统盘和数据盘分开。
+
```

Comment:
Better to add number of node, how many minimum nodes for master and worker .

Reply:
good idea.
I thought about this idea,  standalone need one machine, but not confirm how many machines they really needed in cluster mode, they need to make their own estimate according to their own business situation